readme.md

```diff
@@ -15,3 +15,8 @@ $ django-admin.py startproject \
 $ pip install -r requirements.txt
 $ python <project_name>/manage.py runserver
 ```
+
+# Copy pre-commit hook to git hooks and give it execute permission
+```bash
+$ touch .git/hooks/pre-commit && cp pre-commit.py .git/hooks/pre-commit && chmod -x .git/hooks/pre-commit
```

Comment:
To use the template you don't clone it, u just executes `django-admin.py`, so the correct procedure is:

```bash
$ git init
$ cp pre-commit.py .git/hooks/pre-commit
$ chmod -x .git/hooks/pre-commit
```

You don't need the touch, and I think it's better to a command per line.